dll/ntdll/ldr/ldrinit.c

```diff
@@ -2135,13 +2136,58 @@ LdrpInitializeProcess(IN PCONTEXT Context,
         if (FreeCurDir) RtlFreeUnicodeString(&CurrentDirectory);
     }
 
-    /* Check if we should look for a .local file 
-        FIXME: Thomas suggested that this check might actually be reversed, we should check this file
-               if the flag is NOT set. */
-    if (ProcessParameters && (ProcessParameters->Flags & RTL_USER_PROCESS_PARAMETERS_LOCAL_DLL_PATH))
+    /* Check if we should look for a .local file */
+    if (ProcessParameters && !(ProcessParameters->Flags & RTL_USER_PROCESS_PARAMETERS_LOCAL_DLL_PATH))
     {
-        /* FIXME */
-        DPRINT1("We don't support .local overrides yet\n");
+        UNICODE_STRING ImagePathName = Peb->ProcessParameters->ImagePathName;
+        WCHAR LocalBuffer[MAX_PATH];
+        UNICODE_STRING DotLocal;
+
+        ULONG RequiredSize;
+        RequiredSize = ImagePathName.Length + LdrpDotLocal.Length + sizeof(WCHAR);
```

Comment:
`sizeof(UNICODE_NULL)`

Reply:
Yeah I was afraid gcc was going to do something weird with that, hence why I used WCHAR.